### ISSUE TITLE
FPS Unlocker causes physics issues

### ISSUE THREAD

**Issue by Cryisaguy** [jaipack17/Nature2D#37]
As the title says, FPS unlocker seems to mess with certain physics features such as friction, gravity and possibly other physics-based things as well. At 60 fps, friction behaves expectedly but setting the fps cap to none causes issues in which friction seems to be instantaneous and also causes gravity to be stronger.

### Reproduction Steps
The FPS Unlocker being used is created by axstin.
Using :ApplyForce() with the fps cap set to none.

Code utilized below:
```lua
local Nature2D = require(script:WaitForChild("Nature2D"))

local Viewport = workspace.CurrentCamera.ViewportSize
local Canvas = script.Parent:FindFirstChild("Canvas")

local Engine = Nature2D.init(script.Parent)
Engine:CreateCanvas(Vector2.new(), Viewport, Canvas)

local Floor = Engine:Create("RigidBody", {
	Object = Canvas.Ground,
	Collidable = true,
	Anchored = true
})

local Object = Engine:Create("RigidBody", {
	Object = Canvas.Object,
	Collidable = true,
	Anchored = false
})


Object:CanRotate(false)
Engine:Start()

local UserInputService = game:GetService("UserInputService")
UserInputService.InputBegan:Connect(function(Input)
	if Input.KeyCode == Enum.KeyCode.E then
		Object:ApplyForce(Vector2.new(50, 0), 1)
	end
	
	if Input.KeyCode == Enum.KeyCode.F then
		Object:ApplyForce(Vector2.new(0, -50), 1)
	end
end)
```

### Expected Behavior
For friction and gravity to behave similar to how they behave at 60 fps.

### Actual Behavior
Friction seemed to be instantaneous instead of being smooth, and gravity seems to be normal.

### Media
Behavior with 60 fps cap:

![60fpscap](https://user-images.githubusercontent.com/100386018/155622403-0c51dbb6-807b-4f74-a2b2-068017318941.gif)

Behavior with no fps cap:

![NoCap](https://user-images.githubusercontent.com/100386018/155622455-505e5bf0-6d20-4f00-873d-8636e8499358.gif)

### Lastly
I'm not sure what else this may possibly affect, but I just wanted to notify incase it also shows other issues that fps unlocker may cause. Sorry if this isn't very informative, it's my first bug report.

**Comment by jaipack17** [comment]
Thank you for reporting! I'll work on it. I think the problem lies within how I am using deltaTime.

**Comment by jaipack17** [comment]
Thank you for the patience! This has been fixed in #38 

Will release v0.7.1 tomorrow after completing the documentation and testing the new updates. If you want to try the fix early though, you can clone the `pr-bug-fixes` branch from this repository. Thanks!
